src/components/InputPreview.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ChangeEvent, useState } from "react";
+import { Button } from "./atomic";
+
+type PreviewInput = {
+  setFile(file: File | null): void;
+};
+
+const InputPreview = ({ setFile }: PreviewInput) => {
+  const [src, setSrc] = useState("/static/DefaultLogo.svg");
+
+  return (
+    <div className="flex items-center">
+      <img alt={src} className="object-cover rounded-md w-20 h-20" src={src} />
+      <div className="w-10" />
+
+      <Button variant="inverted" size="small">
+        <label>
+          <input
+            onChange={async (e: ChangeEvent<HTMLInputElement>) => {
+              const file = e.target.files![0];
+              const url = URL.createObjectURL(file);
+              setFile(file);
```

Comment:
![image](https://user-images.githubusercontent.com/19853022/122145703-99303400-ce0a-11eb-8aa0-f876672acc80.png)

Should only allow image files to be uploaded.